iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/BusAdapter.java

```diff
@@ -32,189 +33,190 @@ protected BusAdapter(final Bus delegate, final MergingViewIndex index) {
     }
 
     @Override
-    public ComponentAdapter getSynchronousComponent() {
-        return getIndex().getComponent(getDelegate().getSynchronousComponent());
+    public Iterable<TwoWindingsTransformer> getTwoWindingsTransformers() {
+        return getTwoWindingsTransformerStream().collect(Collectors.toSet());
     }
 
     @Override
-    public ComponentAdapter getConnectedComponent() {
-        return getIndex().getComponent(getDelegate().getConnectedComponent());
+    public Stream<TwoWindingsTransformer> getTwoWindingsTransformerStream() {
+        return getDelegate().getTwoWindingsTransformerStream().map(getIndex()::getTwoWindingsTransformer);
     }
 
-    // -------------------------------
-    // Simple delegated methods ------
-    // -------------------------------
-
-    // -------------------------------
-    // Not implemented methods -------
-    // -------------------------------
     @Override
-    public VoltageLevelAdapter getVoltageLevel() {
-        throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+    public ComponentAdapter getSynchronousComponent() {
+        return getIndex().getComponent(getDelegate().getSynchronousComponent());
     }
 
     @Override
-    public double getV() {
-        throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+    public ComponentAdapter getConnectedComponent() {
+        return getIndex().getComponent(getDelegate().getConnectedComponent());
     }
 
     @Override
-    public BusAdapter setV(final double v) {
-        throw MergingView.NOT_IMPLEMENTED_EXCEPTION;
+    public Iterable<ThreeWindingsTransformer> getThreeWindingsTransformers() {
+        return getThreeWindingsTransformerStream().collect(Collectors.toList());
```

Comment:
Related to a previous comment: `toList()` vs `toSet()`
The comment works for all methods that return `Iteratable`

Reply:
Replaced by Iterables.transform method usage